src/routes/organizations.js

```diff
@@ -212,7 +213,8 @@ export const getOrg = async (req, res) => {
 			if (!organization) {
 				return handleNotFound(res);
 			}
-
+			//Remove Deleted Services
```

Comment:
@Alfredo-Moreira  instead of doing this, I think we should include/exclude the `is_deleted` field in the query. this means we'll be adjusting the function from `findById` to `find({_id: orgId, isDeleted: true | false})`. 
also, if we want to sort results, I think we should utilise the mongo implementation rather than do it ourselves.

Reply:
I can do the first request, do you want to use a request param to determine when getting an org that is deleted and have is_deleted set to false by default?

Regarding the sorting. Unfortunately, MongoDB does not support sorting at a nested array level only at the document level. 

Also this query is to be used by the catalog, it retrieves the full Org document, how do you suggest we remove the soft deleted services?

I tried building a query like 
```
find({
		_id:orgId,
		is_deleted:true | | false,
		'services.is_deleted':false
		});
```
While the is_deleted works, the 'service.is_deleted does not work. Even when used alone:
```
find({
		//_id:orgId,
		'services.is_deleted':true
		})
```
the whole document is returned with values for services.is_deleted equal to both true and false